packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -123,7 +123,8 @@ export async function fetchHelper(
     }, (error) => {
         const online = isOnline();
         const errorText = `${error}`;
-
+        const urlRegex = /((http|https):\/\/([-A-Z0-9.\/]*))/i;
```

Comment:
What about %, #, any other symbols?

Reply:
To illustrate the complexity, look at the Javascript section on this page:  https://urlregex.com/

Maybe a good compromise would be to do this, which just captures everything following http[s]:// until whitespace:

`https?:\/\/[\S]+`

I recommend [www.regex101.com](https://regex101.com/) for testing regex's

Reply:
I use https://regexkit.com/ for testing. But should be same.